webextensions/api/scripting.json

```diff
@@ -7,7 +7,8 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/InjectionTarget",
             "support": {
               "chrome": {
-                "version_added": "88"
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 or later."
```

Comment:
@xeenon 

I noted that Safari is missing scripting.registerContentScripts
```
  .registerContentScripts([
    {
      matches: ['https://*/*', 'http://*/*'],
      id: 'wm-polyfill.js',
      world: 'MAIN',
      js: ['wm-polyfill.js'],
      runAt: 'document_start'
    }
  ])
```

It's currently the only way to inject a polyfill in the Main context at document_start

Reply:
Yes, `registerContentScripts` was added to Chrome after Safari shipped v3 support. It is on our radar too add.

Reply:
@xeenon 
Great to hear! :) 
Thanks for the info